### PR TITLE
WIP preferences: the next step after you apply is a brief interview with your county

### PR DESCRIPTION
**Pull request by dbalatero**
@eahanson here is the WIP for https://www.pivotaltracker.com/story/show/143061055

### CODE REVIEW

File: app/models/app.rb [hunk 1]
```diff
@@ -5,6 +5,11 @@ class App < ApplicationRecord
 
   delegate :full_name, to: :user
 
+  PREFERENCES_FOR_INTERVIEW = [
```
Comment (dbalatero) [inline app/models/app.rb]: @eahanson for some reason these are rendering as:

screenshot 2017-04-06 14 37 54

**Comment by jessieay** [comment]
@eahanson is this still in progress?

**Comment by eahanson** [comment]
Nope, this can be trashed. This is the first time I've even looked at the PRs for this project :)

**Comment by jessieay** [comment]
Great! Closing 😉

**Pull request closed without merging**